package.json

```diff
@@ -29,10 +29,12 @@
     }
   },
   "dependencies": {
+    "@carbon/icons-react": "^0.0.1-alpha.12",
```

Comment:
We already have this as a peer dep, is there a reason to duplicate into dep?

Reply:
I removed and it worked. I missed the peerdep/